Cargo.toml

```diff
@@ -3,6 +3,9 @@ name = "minsql"
 version = "0.1.0"
 authors = ["Daniel Valdivia <daniel@minio.io>"]
```

Comment:
This needs to change to dev@minio.io like all our projects @dvaldivia

Reply:
Changed to `"MinIO <dev@minio.io>"`